docs/conf.py

```diff
@@ -57,8 +57,8 @@
 #
 # The short X.Y version.
 # The full version, including alpha/beta/rc tags.
-# TODO update version
-release = '1.0.0'
+version = '1.0.0'
```

Comment:
Will this now automatically be changed by bump2version?

Reply:
yep, by https://github.com/eWaterCycle/ewatercycle/blob/153-version-bump/.bumpversion.cfg config.

You try iit out by doing 
```shell
bump2version major
```

Reply:
Check, nice!